polaris-react/src/components/LegacyFilters/components/ConnectedFilterControl/ConnectedFilterControl.scss

```diff
@@ -24,6 +24,12 @@
       border-bottom-right-radius: var(--p-border-radius-200);
     }
   }
+
+  &:has(.MoreFiltersButtonContainer.onlyButtonVisible) {
+    + * {
+      margin-left: var(--p-space-200);
```

Comment:
@alex-page addressing [your comment ](https://github.com/Shopify/polaris/pull/11294#discussion_r1419643326) here and setting the spacing on the sibling element

Reply:
I think `:has` is controversial with browser support 🤦‍♂️

Reply:
We wanted to use `:where()` with the se23 flag to flatten the specificity. Both are disallowed per our minimum Safari on iOS version of v13.